packages/storybook/src/ui/organisms/table/table.stories.tsx

```diff
@@ -1,17 +1,179 @@
 import { Story } from '@storybook/react'
 import Table from '@ltht-react/table'
+import {
+  KeyStringValuePair,
+  Maybe,
+  QuestionnaireItem,
+  QuestionnaireResponse,
+  QuestionnaireResponseItem,
+} from '@ltht-react/types'
+import { Header, TableData } from '@ltht-react/table/lib/atoms/table'
+import { answerText, partialDateTimeText } from '@ltht-react/utils'
 import { summaryDefinition, summaryRecordsList } from './table.fixtures'
 import { mockSummaryDefinition, mockSummaryRecordsList } from './table.mockdata'
 
-export const VerticalTable: Story = () => <Table definition={summaryDefinition} records={summaryRecordsList} />
+const MapQuestionnaireObjectsToVerticalTableData = (
```

Comment:
I don't think we should remove the QuestionnaireTable molecule entirely, or move this mapping logic into the story. 
Since the convention in this repo is for components to accept FHIR resources as the parameters I think we have to stick to it (as much as I may personally dislike it :P). The reason I wanted to make the Table component generic is so we have a separation of responsibilities in case we ever switch out react-table for something else. Equally, providing the generic mapper is a nice feature to provide but we still have to provide at least one component that accepts the definitions/records and maps them itself.

Reply:
got it